packages/codegen-ui-react/lib/forms/form-renderer-helper/relationship.ts

```diff
@@ -168,3 +168,55 @@ export const buildManyToManyRelationshipCreateStatements = (
     ),
   ];
 };
+
+export const buildGetRelationshipModels = (fieldName: string, recordName: string) => {
+  return [
+    factory.createVariableStatement(
+      undefined,
+      factory.createVariableDeclarationList(
+        [
+          factory.createVariableDeclaration(
+            factory.createIdentifier('getRelationshipModel'),
+            undefined,
+            undefined,
+            factory.createArrowFunction(
+              [factory.createModifier(SyntaxKind.AsyncKeyword)],
+              undefined,
+              [],
+              undefined,
+              factory.createToken(SyntaxKind.EqualsGreaterThanToken),
+              factory.createBlock(
+                [
+                  factory.createIfStatement(
+                    factory.createIdentifier(recordName),
+                    factory.createBlock(
+                      [
+                        factory.createExpressionStatement(
+                          factory.createCallExpression(factory.createIdentifier(`set${fieldName}`), undefined, [
```

Comment:
It would be best to use the sanitized name instead of `fieldName`. I think there may be a helper to create a setState expression. It might be worth checking out how we're rendering `set___` in other places